run-local.sh

```
@@ -0,0 +1,77 @@
+echo "API: http://localhost:8000/admin"
+echo "UI: http://localhost:3000"
+echo "PGAdmin: http://localhost:5050"
+echo "================================================"
+echo "Press Ctrl+C to stop all services"
+
+# Handle graceful shutdown
+function cleanup {
+    echo "Shutting down services..."
+    kill $API_PID
+    kill $UI_PID
+    cd $(dirname "$0") && docker-compose -f docker-compose.local.yml down
```

Shellcheck (0.10.0)

[warning] Quote this to prevent word splitting.

(SC2046)